unittests/InstructionCountCI/x87.json

```diff
@@ -17815,6 +17815,40 @@
         "bic w20, w21, w20",
         "strb w20, [x28, #1298]"
       ]
+    },
+    "Multiple fld/fst": {
+      "x86InstructionCount": 4,
+      "ExpectedInstructionCount": 22,
+      "x86Insts": [
+        "fld     dword [ebp+16380]",
```

Comment:
For the two 64-bit addressing variants of this in x87.json and x86_f64.json would you what the address modes to be 64-bit so we're testing this instead of the 32-bit zero-extend implementation? I think that would matter more
```
        "fld     dword [rbp+16380]",
        "fstp    dword [rax-0x4]",
        "fld     dword [rbp-0x8]",
        "fstp    dword [rax+16370]"
```

Reply:
I have changed this as suggested. Thanks.